userspace/libsinsp/examples/test.cpp

```diff
@@ -467,6 +536,8 @@ void formatted_dump(sinsp&, sinsp_evt* ev) {
 	} else if(ev->get_category() == EC_NET || ev->get_category() == EC_IO_READ ||
 	          ev->get_category() == EC_IO_WRITE) {
 		net_formatter->tostring(ev, output);
+	} else if(ev->get_info()->category & EC_PLUGIN) {
```

Comment:
We can't use `ev->get_category()` because that only works for syscall-based events.